components/Header.js

```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+const HeaderContainer = styled.div`
+  display: flex;
+  justify-content: space-evenly;
```

Comment:
I am not sure if it is really useful to work with flexbox when you use positioning to position one of two flex items differently. 👀 You should also add `position: relative` to the `HeaderContainer` to position your Logo dependent from your Header and not from your body.

Reply:
You did not reply to this comment. Did you try to change something?